lib/ReactViewModels/ViewState.ts

```diff
@@ -713,7 +721,12 @@ export default class ViewState {
 
 interface Tool {
   toolName: string;
-  getToolComponent: () => React.ComponentType | Promise<React.ComponentType>;
+  getToolComponent: () =>
+    | React.ComponentType
+    | Promise<React.ComponentType>
+    | React.ReactNode
+    | Promise<React.ReactNode>;
```

Comment:
I don't think we want to accept ReactNode here. ReactNode includes `null`, or the output of a component render function.

Reply:
I've removed ReactNode